app/assets/stylesheets/replies.scss

```diff
@@ -171,8 +171,8 @@ div.post-subheader { width: 100%; }
   // in self and in nested blockquotes
   &, blockquote {
     // remove paragraph margins at boundaries
-    & > p:first-child { margin-top: 0; }
-    & > p:last-child { margin-bottom: 0; }
+    & > p:first-child { margin-top: 0px; }
+    & > p:last-child { margin-bottom: 0px; }
```

Comment:
Why the `px` here? `0` is usually interpreted fine without units, and adding units is then just superfluous.

Reply:
I think there was a cop that wanted that, and it looks tidier imho so I went with it. I can revert it and turn off the cop tho if you think that's better

Reply:
I think it's better to drop the unit – some other lints seem to agree: <https://github.com/CSSLint/csslint/wiki/Disallow-units-for-zero-values> – but it might be personal preference? I'd prefer to allow dropping the unit, so these changes should probably be reverted too.